vm-setup/roles/v1aX_integration_test/tasks/remediation.yml

```diff
@@ -0,0 +1,127 @@
+---
+  - name: Get the name of the worker node
+    shell: |
+       kubectl get bmh -n "{{ NAMESPACE }}" -o json | jq -r '.items[]
+       | select(.spec.consumerRef.name | contains("{{ CLUSTER_NAME }}-workers"))
+       | .metadata.name'
+    register: worker_node_name
+
+  - name: Define WORKER_NODE variable with worker node name
+    set_fact:
+      WORKER_NODE: "{{ worker_node_name.stdout }}"
+
+  - name: Get the name of the worker libvirt VM
+    set_fact:
+      WORKER_NODE_VM: "node_0"
+    when: WORKER_NODE == "node-0"
+
+  - name: Get the name of the worker libvirt VM
+    set_fact:
+      WORKER_NODE_VM: "node_1"
+    when: WORKER_NODE == "node-1"
+
+  - name: Get the name of the worker libvirt VM
+    set_fact:
+      WORKER_NODE_VM: "node_2"
+    when: WORKER_NODE == "node-2"
+
+  - name: Get the name of the worker libvirt VM
+    set_fact:
+      WORKER_NODE_VM: "node_3"
+    when: WORKER_NODE == "node-3"
```

Comment:
why not using the replace filter, it would far more elegant

Reply:
The problem with replace is that you are required to give the file to be modified. I don't think that you can replace variable itself.

Reply:
I'm talking about the variable filter : "{{ WORKER_NODE | replace('-','_') }}"

Reply:
Done.